wrappers/map.js

```diff
@@ -116,6 +116,12 @@ class Map extends EventEmitter {
     }
 
     this.geojsonLayers[name] = layer
+
+    map.emit('geojson.add', {
+      name: name,
+      layer: layer,
+      config: config
```

Comment:
I think you'll find the `config` object at this point may not be what you're expecting. Some of the layer config will be there, others have been converted to presenter functions that match the way that Leaflet defines its callbacks to create popups or icons or whatever. Not sure how useful that will be to people.

Otherwise, looks good :+1: